src/style.css

```diff
@@ -1 +1,9 @@
-/* styles go here */
+
+fieldset {
```

Comment:
it's better to style classes instead of tag ones

Reply:
not fixed , do not style the tags, use class selectors
![image](https://user-images.githubusercontent.com/122989585/233176205-fcc174ca-c8af-42b7-87c8-f0464f44141b.png)